indexer/main.go

```diff
@@ -2,59 +2,166 @@ package main
 
 import (
 	"bytes"
+	"fmt"
 	"github.com/PuerkitoBio/goquery"
+	"github.com/skeswa/gophr/common"
 	"io/ioutil"
 	"log"
 	"strings"
+	"sync"
+	"time"
 )
 
+type GoPackage struct {
+	Description    string
+	GitHubURL      string
+	GoDocURL       string
+	IndexTime      string
+	HttpStatusCode int
```

Comment:
Do we want to include packages that are 404 on github? If so we can store an identifier here? Might be nice to keep them in cassandra/elastisearch so we can tell if a package no longer exists without having to make a request